src/components/form/dt-connection/dt-connection.js

```diff
@@ -183,6 +188,19 @@ export class DtConnection extends DtTags {
     return this.filteredOptions;
   }
 
+  _validateRequired() {
+    const { value } = this;
+
+    if (this.required && value) {
```

Comment:
If value is null/undefined and required is true, this won't trigger a required message